Makefile.toml

```diff
@@ -69,7 +69,7 @@ dependencies = [
 description = "Run all tests for the CI 'features' job"
 category = "CI"
 dependencies = [
-    "build-all-features",
+    "check-all-features",
```

Comment:
calling this change out: I think I'll leave it in since features is one of our slowest jobs and there's almost nothing that will be caught by build that won't be caught by check

Reply:
Oh cool, there didn't used to be "check-all-features" but it looks like this was added since we started using this crate. This is a good time to make the change.